cockroachcloud/client-certs-dedicated.md

```diff
@@ -12,7 +12,7 @@ SQL clients may authenticate to {{ site.data.products.dedicated }} clusters usin
 Cockroach Labs recommends using single sign on (SSO) for authentication of human users where possible, and JWT or security certificate for authentication of your application users.
 {{site.data.alerts.end}}
 
-This page describes how to administer the cluster's certificate authority (CA) certificate and authenticate to a cluster using client certificates.
+This page describes how to administer [public key infrastructure (PKI)](../{{site.versions["stable"]}}/security-reference/transport-layer-security.html) for a {{ site.data.products.dedicated }} cluster, using [Hashicorp Vault PKI Secrets Engine](../{{site.versions["stable"]}}/hashicorp-integration.html).
```

Comment:
Please verify how HashiCorp refers to itself and update the term in the dictionary if necessary.

Reply:
That is consistent with Hashi's language. Is there a discrepancy you see in the rhetoric, or... what? I'm not sure what your suggestion is. By "dictionary" I take it you mean glossary? What glossary term are you suggesting I update? Sorry if I'm being dense here but your comment is not clear to me.
https://www.hashicorp.com/products/vault/pki-with-vault

Reply:
By "dictionary" I meant add it to the Vale terms in `.vale/Vocab/CockroachDB/accept.txt` so that you don't have a terminology error in the PR. It looks like we have "HashiCorp" in the acceptable term list, but not "Hashicorp" (lowercase "c") in the link text.